modules/govuk/manifests/node/s_draft_frontend.pp

```diff
@@ -25,4 +25,12 @@
       'PLEK_SERVICE_SEARCH_URI': value => "https://search.${app_domain}";
     }
   }
+
+  if $::aws_migration {
+    $app_domain_internal = hiera('app_domain_internal')
+
+    govuk_envvar {
+      'PLEK_SERVICE_CONTENT_STORE_URI': value => "https://content-store.${app_domain_internal}";
```

Comment:
Should this speak to draft-content-store?

Reply:
There is a host prefix that gets appended internally

Reply:
Ah, yes of course! This is all so clear.